source/client_channel_handler.c

```diff
@@ -199,6 +199,8 @@ static int s_packet_handler_connack(
             (void *)connection);
```

Comment:
Should we add to the trace log? e.g. "invoking on_connection_complete and on_connection_success callbacks"